lib/rules/require-presentational-children.js

```diff
@@ -62,11 +62,13 @@ const NON_SEMANTIC_TAGS = new Set([
   'strike',
   'tt',
   'u',
+  'i',
+  'svg',
```

Comment:
Is updating this list a breaking change? Either way, let's make any changes to the list in a separate PR from adding the configuration option.